From: Debian CLI Applications Team <pkg-cli-apps-team@lists.alioth.debian.org>
Date: Sat, 26 May 2012 12:43:20 +0200
Subject: 000-fix_build_system

@DPATCH@
---
 Makefile   |   16 ++++++++--------
 NAnt.build |   18 ++++++++++--------
 2 files changed, 18 insertions(+), 16 deletions(-)

diff --git a/Makefile b/Makefile
index 7e1dc47..fad13a2 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,8 @@
 # NAnt make makefile for *nix
-MONO=mono
-MCS=gmcs
+MONO=cli
+MCS=mono-csc
 RESGEN=resgen
-TARGET=mono-2.0
+TARGET=mono-4.0
 
 # Contains a list of acceptable targets used to build NAnt
 VALID_TARGETS := mono-2.0 mono-3.5 mono-4.0 net-2.0 net-3.5 net-4.0
@@ -22,13 +22,13 @@ PLATFORM_REFERENCES = \
 endif
 endif
 
-ifeq ($(MONO),mono)
+#ifeq ($(MONO),mono)
 FRAMEWORK_DIR = mono
 DEFINE = MONO
-else
-FRAMEWORK_DIR = net
-DEFINE = NET
-endif
+#else
+#FRAMEWORK_DIR = net
+#DEFINE = NET
+#endif
 
 # Validates TARGET var. If the value of TARGET exists
 # in VALID_TARGETS array, SELECTED_TARGET will contain
diff --git a/NAnt.build b/NAnt.build
index 5f02491..bccb09a 100644
--- a/NAnt.build
+++ b/NAnt.build
@@ -815,12 +815,13 @@
     </target>
 
     <target name="install-linux" depends="build" if="${platform::is-unix()}">
-        <property name="install.prefix" value="/usr/local" />
+        <property name="install.prefix" value="/usr" />
         <if test="${property::exists('prefix') and string::get-length(prefix) != 0}">
             <property name="install.prefix" value="${prefix}" />
         </if>
         <property name="install.share" value="${path::combine(install.prefix, 'share')}" />
-        <property name="install.nant" value="${path::combine (install.share, 'NAnt')}" />
+        <property name="install.libdir" value="${path::combine(install.prefix, 'lib')}" />
+        <property name="install.nant" value="${path::combine (install.libdir, 'NAnt')}" />
 
         <!-- support staged installs -->
         <property name="install.destdir" value="" />
@@ -834,17 +835,18 @@
         </if>
         
         <property name="install.bindir" value="${install.destdir + path::combine(install.prefix, 'bin')}" />
-        <property name="install.pkgconfigdir" value="${install.destdir + path::combine(install.prefix, 'lib/pkgconfig')}" />
+        <property name="install.pkgconfigdir" value="${install.destdir + path::combine(install.libdir, 'pkgconfig')}" />
         <property name="install.copylocation" value="${install.destdir + install.nant}" />
 
         <!-- perform (staged) install -->
         <echo message="Installing NAnt to '${install.copylocation}' ..." />
         <copy todir="${install.copylocation}" overwrite="true">
-            <fileset basedir="${build.dir}">
-                <include name="bin/**/*" />
+            <fileset basedir="${path::combine(build.dir, 'bin')}">
+                <include name="*" />
+                <include name="extensions/**/**/*" />
 
-                <exclude name="bin/nant.tests.config" />
-                <exclude name="bin/*.Tests.*" />
+                <exclude name="nant.tests.config" />
+                <exclude name="*.Tests.*" />
             </fileset>
         </copy>
 
@@ -874,7 +876,7 @@
             the wrapper is created
         -->
         <echo file="${wrapper}" append="false">#!/bin/sh${environment::newline()}</echo>
-        <echo file="${wrapper}" append="true">exec ${path::combine(install.prefix, 'bin')}/mono ${path::combine(install.nant, 'bin')}/NAnt.exe "$@"</echo>
+        <echo file="${wrapper}" append="true">exec ${path::combine(install.prefix, 'bin')}/cli ${install.nant}/NAnt.exe "$@"</echo>
         <exec program="chmod" commandline="a+x ${path::combine(install.bindir, 'nant')}" />
     </target>
 </project>
-- 
