From: Julian Taylor <jtaylor.debian@googlemail.com>
Date: Sat, 26 May 2012 13:37:35 +0200
Subject: lowercase docdir

---
 NAnt.build |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/NAnt.build b/NAnt.build
index cf47759..631a0b3 100644
--- a/NAnt.build
+++ b/NAnt.build
@@ -823,9 +823,9 @@
             <property name="install.destdir" value="${destdir}" />
         </if>
         
-        <property name="install.docdir" value="${install.destdir + path::combine(path::combine(install.share, 'doc'), 'NAnt')}" />
+        <property name="install.docdir" value="${install.destdir + path::combine(path::combine(install.share, 'doc'), 'nant')}" />
         <if test="${property::exists('doc.prefix') and string::get-length(doc.prefix) != 0}">
-            <property name="install.docdir" value="${install.destdir + path::combine(doc.prefix, 'NAnt')}" />
+            <property name="install.docdir" value="${install.destdir + path::combine(doc.prefix, 'nant')}" />
         </if>
         
         <property name="install.bindir" value="${install.destdir + path::combine(install.prefix, 'bin')}" />
-- 
